docs/Configuration.md

```diff
@@ -0,0 +1,22 @@
+# Configuration
+
+The GUI can be configured by changing the editing the file `assets/app.config.json`.
+
+The supported configuration items are the following.
+
+* `"apiEndpoint"`: The URL to use to contact the API, default `"/api"`.
+* `"defaultLanguage"`: The default language of the GUI, defined as a two letters
+  code, default `"en"`. Accepted values: `"da"`, `"en"`, `"fi"`, `"fr"`,
+  "`nb"`, `"sv"`.
```

Comment:
Can the `defaultLanguage` really be a language outside the `enabledLanguages`?

Reply:
No only the languages defined in `enabledLanguages`, I should word that differently.